modules/object-cache/persistent-object-cache-health-check/load.php

```diff
@@ -0,0 +1,251 @@
+			esc_url( $action_url ),
+			__( 'Learn more about persistent object caching.', 'performance-lab' ),
+			/* translators: Accessibility text. */
+			__( '(opens in a new tab)', 'performance-lab' )
+		),
+	);
+
+	if ( wp_using_ext_object_cache() ) {
+		return $result;
+	}
+
+	/**
+	 * Filter whether to suggest using a persistent object cache.
+	 *
+	 * Plugin and theme authors should NOT use this filter to discourage the use of an object cache.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param bool $suggest Whether to suggest using a persistent object cache.
+	 */
+	if ( ! apply_filters( 'perflab_oc_site_status_suggest_persistent_object_cache', false ) ) {
+		$result['label'] = __( 'A persistent object cache is not required', 'performance-lab' );
+
+		return $result;
+	}
+
+	$available_services = perflab_oc_health_available_object_cache_services();
+
+	$notes = __( 'Speak to your web host about what persistent object caches are available and how to enable them.', 'performance-lab' );
+
+	if ( ! empty( $available_services ) ) {
+		$notes .= ' ' . sprintf(
+			/* translators: Available object caching services. */
+			__( 'Your host appears to support the following object caching services: %s.', 'performance-lab' ),
+			implode( ', ', $available_services )
+		);
+	}
+
+	/**
+	 * Filter the second paragraph of the health check's description
+	 * when suggesting the use of a persistent object cache.
+	 *
+	 * Hosts may want to replace the notes to recommend their preferred object caching solution.
+	 *
+	 * Plugin authors may want to append notes (not replace) on why object caching is recommended for their plugin.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param string $notes              The notes appended to the health check description.
+	 * @param array  $available_services The list of available persistent object cache services.
+	 */
+	$notes = apply_filters( 'perflab_oc_site_status_persistent_object_cache_notes', $notes, $available_services );
+
+	$result['status']         = 'recommended';
+	$result['label']          = __( 'You should use a persistent object cache', 'performance-lab' );
+	$result['badge']['color'] = 'orange';
+	$result['description']   .= sprintf(
+		'<p>%s</p>',
+		wp_kses(
+			$notes,
+			array(
+				'a'      => array( 'href' => true ),
+				'code'   => true,
+				'em'     => true,
+				'strong' => true,
+			)
+		)
+	);
+
+	return $result;
+}
+
+/**
+ * Callback for `site_status_persistent_object_cache` filter.
+ *
+ * Determines whether to suggest using a persistent object cache.
+ *
+ * @since 1.0.0
+ *
+ * @param mixed $should_suggest Whether to suggest using a persistent object cache.
+ * @return bool Whether to suggest using a persistent object cache.
+ */
+function perflab_oc_health_should_persistent_object_cache( $should_suggest ) {
+	global $wpdb;
+
+	// Bypass expensive calls if `$should_suggest` was already set truthy.
+	if ( $should_suggest ) {
+		return true;
+	}
```

Comment:
Based on my above comment, let's make this a simple "check" function instead of a filter callback.
```suggestion
/**
 * Determines whether to suggest using a persistent object cache.
 *
 * @since 1.0.0
 *
 * @return bool Whether to suggest using a persistent object cache.
 */
function perflab_oc_health_should_persistent_object_cache() {
	global $wpdb;
```